cmd/run.go

```diff
@@ -107,6 +109,34 @@ a commandline interface for interacting with it.`,
 			Left:  func() string { return "    init" },
 		}
 
+		// Load plugins
+		cliConf, err := getConfig(cmd.Flags())
+		if err != nil {
+			return err
+		}
+
+		plugins := []plugin.JavaScriptPlugin{}
+		for _, pluginPath := range cliConf.Plugins {
+			jsPlugin, err := lib.LoadJavaScriptPlugin(pluginPath)
+			if err != nil {
+				return err
+			}
+
+			// Run plugin setup and add it to the module tree
+			if err = jsPlugin.Setup(); err != nil {
+				return err
```

Comment:
I'm not sure we should just return without tearing down any other potential plugins that succeeded. Might be a TODO.

Reply:
You could've possibly `defer`ed the `teardown` as it is currently run at the end of this same function ...  which definitely needs some refactoring ...